cli/tests/integration/lsp_tests.rs

```diff
@@ -490,6 +493,7 @@ fn lsp_import_assertions() {
       }),
     )
     .unwrap();
+  handle_configuration_request(&mut client);
```

Comment:
This test needed to read the configuration request for the `.js` file. Previously it was only reading it for the `.json` file in the `did_open` call below.